packaging/test_irods_client_rest_cpp.py

```diff
@@ -190,6 +191,98 @@ def test_put_configuration(self):
         os.remove(file1)
         os.remove(file2)
 
+    def test_meta_data_object(self):
+        data_object = 'data_object'
+        token = irods_rest.authenticate('rods', 'rods', 'native')
+
+        with session.make_session_for_existing_admin() as admin:
+            try:
+                path = os.path.join(admin.home_collection, data_object)
+                admin.assert_icommand(['itouch', path])
+                cmds = construct_meta_ops_for_target(path, 'data_object')
+                res = irods_rest.meta(
+                    token,
+                    cmds
+                )
+                self.assertEqual(res, '')
+            finally:
+                admin.run_icommand(['irm', path])
+
+    def test_meta_collection(self):
+        collection = 'collections'
+        token = irods_rest.authenticate('rods', 'rods', 'native')
+
+        with session.make_session_for_existing_admin() as admin:
+            try:
+                path = os.path.join(admin.home_collection, collection)
+                admin.assert_icommand(['imkdir', path])
```

Comment:
Could use the admin's home collection.

Not required, but just another option.